test/fixtures/core.scale/label-offset-vertical-axes.json

```diff
@@ -0,0 +1,40 @@
+{
+    "tolerance": "0.008",
```

Comment:
@simonbrunel `0.008` is indeed too high. But the test does not fail (which it should) even at `0.004` if I change `"labelOffset"` from `25` to `20` for short labels like `"1", "2", "3", "4"`.